src/test/groovy/com/causecode/user/UserManagementServiceSpec.groovy

```diff
@@ -14,6 +16,14 @@ import spock.lang.Specification
 @Mock([User, Role, UserRole, SpringSecurityService])
 class UserManagementServiceSpec extends Specification {
 
+    /*
+     * Issues raised in spockframework repo:
+     *
+     * 1. https://github.com/spockframework/spock/issues/491
+     * 2. https://github.com/spockframework/spock/issues/783
+     */
+    final static Logger log = LoggerFactory.getLogger(UserManagementServiceSpec)
```

Comment:
This codeclimate issue needs to be suppressed. Let's have a discussion while reviewing PR.

Reply:
Yes